site/package.json

```diff
@@ -12,6 +12,7 @@
     "href-checker": "etefera/href-checker#docsify",
     "jest": "^27.0.4",
     "markdownlint-cli": "^0.23.2",
-    "postcss-cli": "^5.0.1"
+    "postcss-cli": "^5.0.1",
+    "super-simple-sitemap-generator": "etefera/super-simple-sitemap-generator"
```

Comment:
why is this pointing to `etefera/...` ?

Reply:
Currently hosted version of the package has an exit code of `1` instead of `0`. Forked and modified it to exit appropriately until it gets merged back into the base repo.

Reply:
ok.